build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -42,6 +43,17 @@
     <PropertyGroup>
       <SdkVersion>@(BuildVersion ->'%(NugetVersion)')</SdkVersion>
       <SdkBrandName>Microsoft .NET Core 1.0.0 - SDK Preview 2</SdkBrandName>
+
+      <ArtifactNameSdk>dotnet-sdk</ArtifactNameSdk>
```

Comment:
@weshaggard @eerhardt 

I've moved these to properties but having properties that we can only differentiate in the global property bag with a prefix as part of the name feels more cluttered. 

Is this the recommended pattern? Are there cleaner ways to go about this?